Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -64,6 +64,28 @@
 FETCH_QUERY = RAW_QUERY or FETCH_QUERY_PARM
 
 
+def get_value_by_dot_notation(dictionary, key):
+    """
+    Get dictionary value by key using dot notation.
+
+    Args:
+        dictionary (dict): The dictionary to search within.
+        key (str): The key in dot notation.
+
+    Returns:
+        The value corresponding to the key if found, otherwise None.
+    """
+    value = dictionary
+    demisto.debug('Trying to get value by dot notation')
+    for k in key.split('.'):
+        if isinstance(value, dict):
+            value = value.get(k)
+        else:
+            demisto.debug(f'Last value is not a dict, returning None. {value=}')
```

Comment:
can it be something else whenever we're using the method?

Reply:
the purpose of it is to be on the sage side, as we don't have control on what the customer sets as key and what is returned from elastic.